lib/util/git.js

```diff
@@ -7,6 +7,12 @@ const util = require('util');
 const exec = require('./exec');
 const promise = require('./promise');
 
+const gitTagSuffixSymbolsStr = '^{}';
```

Comment:
I've tried trying to google for this and finally succeeded. It's a [gitrevisions](https://www.kernel.org/pub/software/scm/git/docs/gitrevisions.html) thing.

> \<rev>^{\<type>}, e.g. v0.99.8^{commit}
A suffix ^ followed by an object type name enclosed in brace pair means the object could be a tag, and dereference the tag recursively until an object of that type is found or the object cannot be dereferenced anymore (in which case, barf). <rev>^0 is a short-hand for <rev>^{commit}.

> \<rev>^{}, e.g. v0.99.8^{}
A suffix ^ followed by an empty brace pair means the object could be a tag, and dereference the tag recursively until a non-tag object is found.

You can refer to a git tag as the `tag` or `tag^{}` or `tag^{commit}`.